src/interaction/SmoothSquareWell.py

```diff
@@ -83,7 +99,7 @@ class SmoothSquareWell(Potential):
         'The SmoothSquareWell potential.'
         pmiproxydefs = dict(
             cls = 'espressopp.interaction.SmoothSquareWellLocal',
-            pmiproperty = ['epsilon', 'sigma', 'width', 'a']
+            pmiproperty = ['epsilon', 'sigma', 'Lambda', 'a']
```

Comment:
`lambda`

Reply:
lambda wouldn't work because it is a python keyword, unfortunately.

Reply:
True, forgot about that, soo keep like it is